src/logging.c

```diff
@@ -78,31 +87,27 @@ rcutils_ret_t rcutils_logging_initialize_with_allocator(rcutils_allocator_t allo
       g_rcutils_logging_output_format_string[chars_to_copy] = '\0';
     } else {
       if (NULL != ret_str) {
-        fprintf(
-          stderr,
-          "Failed to get output format from env. variable: %s. Using default output format.\n",
-          ret_str);
+        RCUTILS_SET_ERROR_MSG(
+          "Failed to get output format from env. variable. Using default output format.",
```

Comment:
It would be good if this error message could include the `ret_str` in some way.

Reply:
done in https://github.com/ros2/rcutils/pull/65/commits/2d5f39b90bd210975a626f53fae295431da97f3e